### PR TITLE
Contributing information

### LINKED ISSUE #5795: Contributing labels outdated
Contributing information about the labels of the issues and pull request is outdated.

### PR DESCRIPTION
**Pull request by andreugallofre**
**What this PR does / why we need it**:

Change CONTRIBUTING.md to match the actual information on the repository on the lables used for issues and pull request.

**Special notes for your reviewer**:

I'm not sure that the merging conditions of the tags size/L , size/XL and size/XXL are correct because did not found any documentation about it.

This PR closes #5795

**If applicable**:
- [x] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

File: CONTRIBUTING.md
Comment (bacongobbler) [inline CONTRIBUTING.md]: missing period at end of sentence

File: CONTRIBUTING.md
Comment (bacongobbler) [inline CONTRIBUTING.md]: missing period at end of sentence

File: CONTRIBUTING.md
Comment (bacongobbler) [inline CONTRIBUTING.md]: Can you remove "This should be thoroughly tested before merging and always requires 1 approvals"? I feel that the second line (manual validation should be required) covers the first statement, and every PR requires at least 1 approval so its's moot to call that out here.

File: CONTRIBUTING.md
Comment (bacongobbler) [inline CONTRIBUTING.md]: size/L requires 2 approvals.

**Review by bacongobbler — CHANGES_REQUESTED** [review]

**Comment by atsuky** [comment]
@bacongobbler The commits should be squashed? or can i leave it like that in order to have the complete history? 

Thanks!

**Comment by bacongobbler** [comment]
Generally speaking it's better to keep them squashed. Thanks!

**Comment by atsuky** [comment]
Done, commits squased! 👍

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**
